platforms/jvm/language-java/src/test/groovy/org/gradle/api/tasks/compile/CompileOptionsTest.groovy

```diff
@@ -80,7 +80,7 @@ class CompileOptionsTest extends Specification {
 
     def "define"() {
         compileOptions.debug.set(false)
-        compileOptions.fork = false
+        compileOptions.fork.set(false)
```

Comment:
❓ Is `set` needed here?

Reply:
It actually works with `=`, true. It's just not detected by IDE